src/ProgressOnderwijsUtils.Analyzers/.editorconfig

```diff
@@ -0,0 +1,9 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+root = true
+
+# All files
+[*]
+indent_style = space
+
+[*.cs]
+dotnet_diagnostic.RS1004.severity = silent

```

Comment:
We hebben geen overkoelende editorconfig, of deze spreekt de overkoepelende tegen (en zo ja, moeten we niet de gedeelde fixen)?

Reply:
Ik heb het aangepast dat de overkoepelende nu wel wordt gebruikt. En deze setting lijkt me enkel nuttig voor dit project.

Reply:
kun je deze warning inline op de ene plek waar het relevant is fixen?

https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.md klinkt ook alsof het misschien oplosbaar is?

Reply:
Hij is nu helemaal weg ...